src/Parser/DeathKnight/Unholy/Modules/Features/RpPoolingDA.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+
+class RpPoolingDA extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.DARK_ARBITER_TALENT.id);
+  }
+
+  totalRunicPowerPooled = 0;
+  totalDarkArbiterCasts = 0;
+  currentRP = 0;
+  maxRP = 0;
+  
+   on_toPlayer_energize(event){
+	   this.currentRP = event.classResources[0]['amount']/10 || 0;
+	   this.maxRP = event.classResources[0]['max']/10;
+   }
+   
+  on_byPlayer_cast(event){
```

Comment:
I think casts also give you an update of class resources, you might want to take those values into account for a higher accuracy.